value/src/main/java/com/google/auto/value/extension/AutoValueExtension.java

```diff
@@ -82,8 +84,8 @@ public boolean applicable(Context context) {
    * in the inheritance hierarchy.  Only one extension may be the final class, so
    * this should be used sparingly.
    *
-   * @param context The {@link com.google.auto.value.AutoValueExtension.Context} of the code
-   *                generation for this class.
+   * @param context The {@link com.google.auto.value.extension.AutoValueExtension.Context} of the
```

Comment:
I think it's enough to write Context. There will be a link to that type in the javadoc anyway since it's the type of the parameter.